corehq/apps/hqadmin/views.py

```diff
@@ -726,3 +726,28 @@ def _malt_csv_response(month, year):
     query_month = "{year}-{month}-01".format(year=year, month=month)
     queryset = MALTRow.objects.filter(month=query_month)
     return export_as_csv_action(exclude=['id'])(MALTRowAdmin, None, queryset)
+
+
+@require_superuser
+def branches_on_staging(request, template='hqadmin/branches_on_staging.html'):
+    branches = _get_branches_merged_into_autostaging()
+    return render(request, template, {'branches': branches})
+
+
+def _get_branches_merged_into_autostaging():
+    import sh
+    git = sh.git.bake(_tty_out=False)
+    # %p %s is parent hashes + subject of commit message, which will look like:
+    # <merge base> <merge head> Merge <stuff> into autostaging
+    pipe = git.log('origin/master...', grep='Merge .* into autostaging', format='%p %s')
```

Comment:
What does `git log origin/master...` do?  Wouldn't you want to log `HEAD` (assuming this is meant to be viewed on staging)?